### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ryrjanes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
+## Getting Help or Reporting an Issue
+ 
+For any questions regarding this project, or for inquiries about starting a new analytics account, please contact the GDX Analytics Team.
+ 
+## How to Contribute
+ 
+If you would like to contribute to the guide, please see our [CONTRIBUTING](CONTRIBUTING.md) guideleines.
+ 
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
```
Comment (f-w) [inline README.md]: Change this section to 
## Contributors
The GDX Analytics Team are the main contributors to this project and maintain the code.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,35 @@
-## GDX Looker
+## GDX Analytic Looker FOI
```
Comment (danpollock) [inline README.md]: Should be "GDX Analytics Looker FOI"

**Comment by danpollock** [comment]
Looks like we're missing the "## Requirements" section.

File: README.md (hunk 1 above)
Comment (ryrjanes) [reply README.md]: Made the change

File: README.md (hunk 2 above)
Comment (ryrjanes) [reply README.md]: made the change

**Comment by ryrjanes** [comment]
made requested changed. please approve.

**Review by f-w — APPROVED** [review]
OK

**Pull request merged**